Makefile

```diff
@@ -40,8 +39,16 @@ run:
 env:
 	./scripts/bootstrap-virtualenv.sh
 
-lint:
-	${PYTHONLINT} ${PYDIRS}
+lint: listoutdated flake8diff
+
+listoutdated:
+	pip list --outdated --allow-external=cafe,cloudcafe
+
+flake8diff:
+	git diff --patch --no-prefix origin/master | flake8 --diff
```

Comment:
1.  Is there a config file or something that specifies the maximum line length?  `pep8` is 80, and we were using 105 before.

2.  Doesn't `git diff --patch --no-prefix origin/master` mean that if your branch is behind master, it'd also lint all the files that are in master but not in your branch yet?  This shouldn't matter as we move forward and start using this lint style, but for instance if you run `make lint` now (there are a bunch of errors from changes that have been merged to master but not this branch).